framework/log/SyslogTarget.php

```diff
@@ -26,6 +26,10 @@ class SyslogTarget extends Target
      * @var int syslog facility.
      */
     public $facility = LOG_USER;
+    /**
+     * @var int openlog option
+     */
+    public $option = LOG_ODELAY | LOG_PID;
```

Comment:
@cebe, I agree. But PHP5.4 and PHP 5.5 does not allow LOG_ODELAY | LOG_PID. So what do you suggest? Set just one value, or set the default value in the init?